build.gradle

```diff
@@ -116,7 +117,7 @@ subprojects {
 
 ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
-    neo4jVersion = "4.3.0-drop03.0"
+    neo4jVersion = "5.0.0-dev"
```

Comment:
Yes. It's just that we publish it with the `-dev` suffix in TeamCity instead of the SNAPSHOT. For local development we should be overriding this with the env variable `export neo4jVersionOverride=5.0.0-SNAPSHOT`. They were developing overriding that variable anyway. To set it out in IntelliJ one must do this:

https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html

Reply:
This is needed for TeamCity. I've addressed it so we don't have to do manual override locally here: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/2564/commits/8b3af787161d34d5fa854fc2bc6b7ec6f2a042d3